### PR TITLE
TM-4129 remarks read on new structure

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
This reads the new remarks structure and adds in error handling checks.

currently deployed in dev1 - if you want to go test it out over there instead

Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/778)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "indent": ["error", 2, { "ignoredNodes": ["JSXAttribute", "JSXSpreadAttribute"], "SwitchCase": 1 }],
     "react/jsx-indent": [0],
     "react/jsx-indent-props": ["error", 2],
-    "complexity": ["warn", { "max": 32 }],
+    "complexity": ["warn", { "max": 40 }],
```
Comment (M00staff) [inline .eslintrc:74]: 😁

File: src/Components/Agenda/RemarksPill/RemarksPill.jsx [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
 
 const RemarksPill = props => {
-  const { remark, isEditable, updateSelection } = props;
+  const { remark, isEditable, updateSelection, fromAIM } = props;
 
   const getRemarkText = (r) => {
-    if (!isEmpty(r)) {
-      let rText = get(r, 'text');
-      rText = rText.split(' ');
+    const refInserts = get(r, 'remark_inserts') || [];
```
Comment (M00staff) [inline src/Components/Agenda/RemarksPill/RemarksPill.jsx:10]: is there a particular reason y'all use lodash so heavily?

or is it just what they initially started out with and became a habit - I'm pretty sure optional chaining made `get` specifically a bit obsolete and slow comparatively.

Comment (elizabeth-jimenez) [reply src/Components/Agenda/RemarksPill/RemarksPill.jsx:10]: Yep, habit! Good point - I saw what you did over in ur PR and love how it looks too. Time for a new habit ?.

Comment (elizabeth-jimenez) [reply src/Components/Agenda/RemarksPill/RemarksPill.jsx]: Pushed the updates - all the lodash gets I used for this PR have been moved to optional chaining

File: src/sass/_agendaItemMaintenance.scss [hunk 3]
```diff
@@ -52,37 +52,6 @@
...
-  //@media screen and (min-width: 3000px) {
-  //  width: 93vw;
-  //}
-  //@media screen and (min-width: 3300px) {
-  //  width: 94vw;
-  //}
-  //@media screen and (min-width: 4000px) {
-  //  width: 95vw;
-  //}
-  // at 1600 we auto stack, so smaller screen styles should go there
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/sass/_agendaItemMaintenance.scss:85]: Red means it was removed 😅

[the remark this answers is not in the document]
Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:85]: I'm slow all the `//` through me off😂

File: src/sass/_panelMeetingAgendas.scss [hunk 4]
```diff
@@ -188,35 +188,6 @@
...
-        margin-right: 1.5rem;
-        padding: .5em;
-      }
-
-      @each $remark-id, $color in $remarks {
-        .remark-category--#{$remark-id} {
-          background-color: $color;
-        }
-      }
-    }
```
Comment (scott062) [inline src/sass/_panelMeetingAgendas.scss:191]: I assume this was just copied/pasted stylings we already had?

Comment (elizabeth-jimenez) [reply src/sass/_panelMeetingAgendas.scss:191]: exactly

**Review by scott062 — APPROVED** [review]
🦢

**Pull request merged**